{"name": "test_reg_exist_user", "status": "passed", "description": "\n        Тестируется негативный сценарий, когда производится попытка зарегестрировать уже существующего в базе\n        пользователя.\n        Приложение работает корректно и пользователю выводится сообщение об ошибке\n        ", "start": 1590601100979, "stop": 1590601118199, "uuid": "7ee84d9e-b279-4cee-ab1d-6388a6a810b3", "historyId": "f13d0b46e2d15c16461cb19d9576adbc", "testCaseId": "e2c2d27d593446941c8abed86454d1dd", "fullName": "tests.test_ui.TestReg#test_reg_exist_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "TestReg"}, {"name": "host", "value": "MacBook-Air-Roman-2.local"}, {"name": "thread", "value": "20068-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}